### ISSUE TITLE
Midterm evaluation from adk87

### ISSUE THREAD

**Issue by andrewkozma** [ORIE4741/projects#67]
This project's goal is to detect arbitrage opportunities in the stock market. Arbitrage is created when the net present value of a stock and its actual trading price do not match. This allows people to make free money, basically. 

I think this is a very interesting idea; who doesn't want to know when they can make free money. It is important that you test this out over the stocks of multiple companies in order to generalize the model. One question I do have is if you have previous examples of arbitrage given past stock prices. This would be useful in testing how well your model works. Or, is the point of the project just to create a model of the stock prices? If this is the case, then what you are predicting is the future stock prices, not arbitrage opportunity? 

I am glad that you took into consideration that you needed at least 102 observations to train as many parameters. If this results in a square matrix, then I am curious as to what you used for your training and test set. I didn't see mention of where the test set came from, although you did mention that the model performed poorly on the test set. This definitely will be an issue since your training set is so small. Also, is 102 observations enough to train a model on and there ways you could get more examples of data to increase the size of your dataset?

I am confused as to what the plots are that are in the report. It would be very helpful if the horizontal and vertical axes in the plots were labeled.  Also, I was confused as to how Principal Component Analysis worked. I'm sure we can use things we haven't learned in class, but it would be cool if things we learned in class could be implemented into the project.
